### ISSUE TITLE
Tooltip Custom Feature

### ISSUE THREAD

**Issue by Danny-Tran** [coroo/nova-chartjs#24]
Hello I am trying to hide the legend for my graph, looks like it not working?
this is in the doc 
![image](https://user-images.githubusercontent.com/41171906/74978950-dccb8700-53ea-11ea-8824-b946a78d2c7c.png)
this is what I have in my code
![image](https://user-images.githubusercontent.com/41171906/74979050-0c7a8f00-53eb-11ea-9873-697f28408a85.png)
and this is what the nova-chartjs produce with legend attached
![image](https://user-images.githubusercontent.com/41171906/74979134-392ea680-53eb-11ea-9d9f-d488dfc0b4fc.png)

**Comment by Danny-Tran** [comment]
I upgrade to V0.1.2 and this solve the problem!
I am also wondering Am i able to edit the tooltips like in this doc?
https://www.chartjs.org/docs/latest/configuration/tooltip.html?h=tool

**Comment by coroo** [comment]
Hi @Danny-Tran thank you for your interest for this package. For tooltip, I will try to check it first and let you know again soon.

**Comment by daggemarren** [comment]
What's status for Tooltips? Can't figure out how to pass options or callbacks in current version

https://www.chartjs.org/docs/latest/configuration/tooltip.html?h=tooltips

**Comment by coroo** [comment]
Hi @Danny-Tran & @daggemarren ,

Sorry for take so long for this feature. 
Actually, this feature is on testing stage and If you can help me to check all the tooltips options list, it will be great.

For help me to test, you can check at 
dev-issues-24: 
```
composer require coroowicaksono/chart-js-integration "dev-issues-24"
```

Example Test:
```
(new StackedChart())
        ->title('Revenue')
        ->series(array([
            'barPercentage' => 0.5,
            'label' => 'Product #1',
            'backgroundColor' => '#ffcc5c',
            'data' => [30, 70, 80],
        ],[
            'barPercentage' => 0.5,
            'label' => 'Product #2',
            'backgroundColor' => '#ff6f69',
            'data' => [40, 62, 79],
        ]))
        ->options([
            'xaxis' => [
                'categories' => [ 'Jan', 'Feb', 'Mar' ]       
            ],
            'tooltips' => [
                'backgroundColor' => 'rgba(145, 145, 145, 0.8)',
            ]
        ])
        ->width('1/3'),
```

**Comment by coroo** [comment]
This issue has been solved in [v0.2.4](https://packagist.org/packages/coroowicaksono/chart-js-integration).

Please see documentation in
- [Tooltip](https://coroo.github.io/nova-chartjs/#/?id=tooltips)
- [Tooltip Callback](https://coroo.github.io/nova-chartjs/#/?id=tooltip-callback)
